pyproject.toml

```diff
@@ -4,7 +4,9 @@ version = "0.1.0"
 description = "Add your description here"
 readme = "README.md"
 requires-python = ">=3.11"
-dependencies = []
+dependencies = [
+    "cachetools>=5.5.1",
```

Comment:
I don't want to add a dependency on any external project. If we _really had to_, we could use it as an optional dependency only within the fsspec integration, but I'd prefer to avoid any external dependency at all.